src/BehaviorsSDKManaged/ManagedUnitTests/TestVisualTreeHelper.cs

```diff
@@ -10,19 +10,19 @@ namespace ManagedUnitTests
     /// </summary>
     internal class TestVisualTreeHelper : IVisualTreeHelper
     {
-        private Dictionary<DependencyObject, DependencyObject> parents = new Dictionary<DependencyObject, DependencyObject>();
+        private Dictionary<DependencyObject, DependencyObject> _parents = new Dictionary<DependencyObject, DependencyObject>();
 
         public void AddChild(DependencyObject parent, DependencyObject child)
         {
-            this.parents[child] = parent;
+            this._parents[child] = parent;
```

Comment:
I wonder if we shouldn't also just drop all the `this.` bits?

Reply:
We discussed using the underscore, but not removing this. I would rather remove all instances of `this.` Whether access a private member variable, a public property, or a private method. There isn't a need for `this.`. However, this PR is focused on using the underscore. If we are all in agreement to remove `this.` I can add that change as well!

Reply:
I vote for  the removal of .this

Reply:
I second the "no more .this" motion! 🙂